sycl/include/CL/sycl/queue.hpp

```diff
@@ -122,6 +122,89 @@ class queue {
     });
   }
 
+  // single_task version with a kernel represented as a lambda.
+  template <typename KernelName = detail::auto_name, typename KernelType>
+  event single_task(KernelType KernelFunc) {
+    return submit([&](handler &cgh) {
+      cgh.template single_task<KernelName, KernelType>(KernelFunc);
+    });
+  }
+
+  template <typename KernelName = detail::auto_name, typename KernelType>
+  event single_task(std::vector<event> Events, KernelType KernelFunc) {
+    return submit([&](handler &cgh) {
```

Comment:
```suggestion
    return submit([&](handler &CGH) {
```
The same comment to other functions